std/experimental/allocator/typed.d

```diff
@@ -144,7 +144,8 @@ struct TypedAllocator(PrimaryAllocator, Policies...)
     // state {
     static if (stateSize!PrimaryAllocator) private PrimaryAllocator primary;
     else alias primary = PrimaryAllocator.instance;
-    private Tuple!(Stride2!(Policies[1 .. $])) extras;
+    static if (Policies.length > 0)
+        private Tuple!(Stride2!(Policies[1 .. $])) extras;
```

Comment:
You should probably `static assert` in the `else` case or something similar as `extras` is referenced below and it won't be defined if the `static if` fails.